02 Local Platform/09 Datasets/04 Downloading Data/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "metadata",
+    "values": {
+        "description": "Download data files for backtesting, researching, and live trading.",
```

Comment:
Extend this to note that the data files come from the QuantConnect Dataset Market.